__tests__/Port.test.js

```diff
@@ -0,0 +1,42 @@
+const Port = require('../src/Port');
+
+describe("constructor", () => {
+    let astapor;
```

Comment:
Why don't you move this variable and the 4 down below to a more global scope, so then all you have to do is to assign the values in the beforeEach methods?